### PR TITLE
Fix temperature indexing error for plot_xs

### PR DESCRIPTION
**Pull request by smharper**
`plot_xs` currently doesn't work if you give it a temperature that isn't in your library.  The code tries to find the nearest temperature, but it runs into an error where it compares floats to strings.  This PR fixes that.

### CODE REVIEW

File: openmc/plotter.py [hunk 1]
```diff
@@ -359,16 +359,9 @@ def _calculate_cexs_nuclide(this, types, temperature=294., sab_name=None,
...
-                data_Ts[t] = float(data_Ts[t][:-1])
-            min_delta = float('inf')
-            closest_t = -1
-            for t in data_Ts:
-                if abs(data_Ts[t] - T) < min_delta:
-                    closest_t = t
-            nucT = "{}K".format(int(round(data_Ts[closest_t])))
+            delta_T = np.array(nuc.kTs) - T*openmc.data.K_BOLTZMANN
+            closest_index = np.argmin(np.abs(delta_T))
+            nucT = nuc.temperatures[closest_index]
```
Comment (paulromano) [inline openmc/plotter.py]: Looks like this same logic is used below for S(a,b) as well. Can you change it there too?

File: openmc/plotter.py [hunk 2]
```diff
@@ -359,16 +359,9 @@ def _calculate_cexs_nuclide(this, types, temperature=294., sab_name=None,
...
-            for t in range(len(data_Ts)):
-                # Take off the "K" and convert to a float
-                data_Ts[t] = float(data_Ts[t][:-1])
-            min_delta = float('inf')
-            closest_t = -1
-            for t in data_Ts:
-                if abs(data_Ts[t] - T) < min_delta:
-                    closest_t = t
-            nucT = "{}K".format(int(round(data_Ts[closest_t])))
+            delta_T = np.array(nuc.kTs) - T*openmc.data.K_BOLTZMANN
```
Comment (nelsonag) [inline openmc/plotter.py]: space around the `*` operator

**Comment by nelsonag** [comment]
My browser is acting up so I'm not sure if it was sent or not, but I left one additional comment regarding adding spaces around the * operator. That's it from me

**Comment by paulromano** [comment]
Thanks @smharper!

**Pull request merged**